src/e2e/controllers.e2e.js

```diff
@@ -37,7 +37,7 @@ beforeAll(async () => {
         headers: { 'Access-Control-Allow-Origin': '*' },
         body: JSON.stringify(generateMockControllerAggregation),
       });
-    } else if (request.method() === 'GET' && request.url().includes('aliases')) {
+    } else if (request.method() === 'POST' && request.postData().includes('aliases')) {
```

Comment:
I'd prefer the JSON payload check to be a bit more specific than just having "aliases" anywhere. Though most likely the test environment won't ever see it anywhere but the expected "data"... and, OK, the adjacent tests aren't any more specific. :-(

The repetition of this pattern so many times seems awkward, too; what's really happening is trying to identify a query from the HTTP method and a "unique" URI or some snippet of JSON payload. This could probably be refactored and cleaned up (and made more robust); but despite griping about it, I'm not going to push you to attempt that now.

Reply:
The JSON payload is just a small mock payload that we return for the purpose of e2e tests and I am not sure but I think that might have been the original reason behind this checks, I can definitely clean this up a little but this is again going to be modified when we have query specific REST APIs

Reply:
What is the reason for the change from `GET` to `POST`?